Makefile.am

```diff
@@ -708,10 +761,11 @@ check-local:
 	@echo "Running unit tests"
 	@vg= ; \
 		test -z "$$VG" || vg="$(VALGRIND)" ; \
+		test -z "$$SUPPRESS" && suppress=$(LSAN_SUPPRESS) ; \
 		f="-v" ; \
 		test "x$$CUFORMAT" = xjunit && f="-x" ; \
 		cd cunit ; \
-		$$vg ./unit $$f ; \
+		$$vg $$suppress ./unit $$f ; \
 		retval=$$? ; \
 		if [ "x$$CUFORMAT" = xjunit ] ; then \
 			$(RM) -rf reports ; mkdir reports ; ./cunit-to-junit.pl ; \
```

Comment:
As an aside, I really hate this rule, and keep vaguely meaning to replace the multi-line embedded shell script with a separate script that we just call with arguments.